examples/person-detection/person_detection.py

```diff
@@ -21,21 +22,23 @@
 !mkdir -p binaries/person_detection && cd binaries/person_detection && wget https://github.com/antmicro/tensorflow-arduino-examples-binaries/raw/master/person_detection/person_detection.ino.elf # fetch prebuilt binaries
 
 # %% [markdown]
-"""## Take a photo"""
+"""
+## Take a photo
+To change the selected option, rerun the cell
+# """
 
 # %%
-from IPython.display import Image
-photo = image.take_photo()
-display(Image(photo))
-
+image.image_options()
```

Comment:
It works better now, but can we make it overwrite the previous image, not stack them? It does not really help

Reply:
also the upload button allows has a strange counter. If you run it twice it shows (2). What is this supposed to be?

Reply:
There is no any handler for a displaying image, so it can't be disabled and images are stacked. I recommend to rerun cell and clear the output.